src/parser/seclang-scanner.ll

```diff
@@ -405,7 +405,7 @@ CONFIG_VALUE_ONLYARGS                   (?i:OnlyArgs)
 CONFIG_VALUE_OFF                        (?i:Off)
 CONFIG_VALUE_ON                         (?i:On)
 CONFIG_VALUE_PARALLEL                   (?i:Parallel|Concurrent)
-CONFIG_VALUE_PATH                       [0-9A-Za-z_\/\.\-\*\:]+
+CONFIG_VALUE_PATH                       [0-9A-Za-z_\/\.\-\*\: \\\(\)]+
```

Comment:
You're in a character class here, no need to escape meta characters.
```suggestion
CONFIG_VALUE_PATH                       (?i:[0-9a-z_/.*: \\()-]+)
```

Reply:
That was the original form, I just added the new characters. Let's see how does it work.

Also good to know that Bison has its own regex engine, not PCRE or other 3rd-party. Theoretically it follows POSIX standard, so I hope this changes won't change the behavior.

Reply:
Unfortunately this does not work, I'm afraid we need to escape the sequences too - see failed tests, like [this](https://github.com/owasp-modsecurity/ModSecurity/actions/runs/16668799425/job/47180283694?pr=3430#step:9:60).

I'm going to revert that commit.